includes/class-coblocks-labs.php

```diff
@@ -115,6 +129,13 @@ public function propagate_settings() {
 			update_option( 'coblocks_site_design_controls_enabled', 0 );
 			update_option( 'coblocks_layout_selector_controls_enabled', 0 );
 		}
+
+		// Site design is disabled when in GSE context.
+		$is_go_se = self::is_go_se();
+
+		if ( $is_go_se ) {
```

Comment:
Nit picking, but I don't think that the variables are necessary here, since they are only used in this method. We can probably drop the two lines where the variables are assigned.

```php
if ( ! self::is_go_active() ) {
	update_option( 'coblocks_site_design_controls_enabled', 0 );
	update_option( 'coblocks_layout_selector_controls_enabled', 0 );
}

if ( self::is_go_se() ) {
	update_option( 'coblocks_site_design_controls_enabled', 0 );
}
```